twoliter/embedded/build.Dockerfile

```diff
@@ -436,11 +438,13 @@ ENV VARIANT=${VARIANT} VERSION_ID=${VERSION_ID} BUILD_ID=${BUILD_ID}
 ARG BYPASS_SOCKET
 ARG OUTPUT_SOCKET
 ARG BUILDER_UID
+ARG KMOD_KITS
 
 USER root
 
 WORKDIR /tmp
 RUN --mount=target=/host \
+    if [[ -n "${KMOD_KITS}" ]]; then \
```

Comment:
See usage of `BUILDSYS_KMOD_KIT` in `Makefile.toml` - if we don't produce a kmod kit then we will need to conditionalize the inclusion in the `repo` task.